packages/shared/routes/setup/ledger/views/AccountIndex.svelte

```diff
@@ -25,6 +34,25 @@
         pollLedgerLegacyStatus()
     })
 
+    function checkNumber(n: number): number {
+        if(!isWithinRange(n))
+            n = Math.min(Math.max(n, min), max)
+
+        return n
+    }
+
+    function isValidNumber(n: number): boolean {
+        return isPositiveInteger(n) && isWithinRange(n)
+    }
+
+    function isPositiveInteger(n: number): boolean {
```

Comment:
@maxwellmattryan maybe we could add some error messages for the validators? I pasted for example `194909e` into the input and its not allowing me to continue but there is no feedback.
![image](https://user-images.githubusercontent.com/3624944/124478963-06017300-dda6-11eb-931e-8c1575fbf43f.png)


Maybe we could use something like we have in `ImportTextfield.svelte`  

![image](https://user-images.githubusercontent.com/3624944/124478839-e2d6c380-dda5-11eb-8cff-d19c7b586538.png)

Reply:
If we add these validators, I think we can remove the `validate` function in `packages/shared/components/inputs/Number.svelte` which forces a min/max